react-events/my-app/src/App.js

```diff
@@ -0,0 +1,13 @@
+import CustomButton from "./CustomButton.js";
+
+function App() {
+  return (
+    <>
+      <CustomButton text="I" color="green" onCustomClick={(text) => alert(`You clicked on the ${text} button!`)} />
```

Comment:
Since all 3 of these handlers are identical, you could extract them into a single `handleCustomClick` function.

Reply:
<img width="819" alt="Screenshot 2023-03-31 at 9 39 28 AM" src="https://user-images.githubusercontent.com/110208589/229179707-8ef8f438-eae9-430a-9614-ee6206e471b1.png">